client-js/README.md

```diff
@@ -38,9 +41,9 @@ npm i casper-erc20-js-client
     [owner]
   );
 
-  await client.putDeploy(deploy);
+  const deployHash = await erc20.putDeploy(deploy);
 
-  const result = await client.nodeClient.waitForDeploy(deploy);
+  const result = await erc20.waitForDeploy(deployHash);
```

Comment:
please remove this part, I think this is responsibility of the developer to handle this - we don't need provide method for this.